docs/3.FormDescriptions.md

```diff
@@ -1,20 +1,20 @@
-# **3. Form Question Descriptions**
+## **3. Form Question Descriptions**
 
-## **Section 1. General Information**
+### **Section 1. General Information**
 
 **A. Researcher Name:** Name of the researcher inputting the dataset in Read the Doc
 
 **B. Project Name:** Choose the name of the project the researcher is using the dataset for.
 
 **C. Dataset Name:** Name of the dataset created by the publisher. Ensure that your dataset name is agnostic to your project name, to ensure that it can be distinguishable from the project names.
```

Comment:
@harnoors mind specifying which one needs the naming convention? I can link the pep8 naming conventions, and if there's any additional requirements for regex checks, I'll add those in too. Or at least the link to them.

Reply:
The folder and file are saved using the dataset name, also there are regex checks in place for anything other than a-z and A-Z is replaced with an underscore.

Reply:
I think it will be sufficient to say that does not include any spaces/special characters/numbers but my checks are there just in case someone forgets to follow these guidelines :-)